proposals/0296-larger-transactions.md

```diff
@@ -0,0 +1,143 @@
+---
+simd: '0296'
+title: Larger Transaction Size
+authors:
+  - jacobcreech
+  - apfitzge
+category: Standard
+type: Core
+status: Review
+created: 2025-05-28
+feature: (fill in with feature key and github tracking issues once accepted)
+---
+
+## Summary
+
+Historically, messages transmitted over the network must not exceed the IPv6 MTU
+size to ensure a fast and reliable network delivery. Solana has used a 
+conservative MTU size of 1280 bytes, which after accounting for some of the
+overhead, leaves a maximum transaction size of 1232 bytes for data in the
+payload.
+
+With QUIC, we can now use a larger transaction size than the original 1280
+bytes. This proposal presents a new transaction size of 4096 bytes.
+
+## Motivation
+
+The current transaction size limit of 1232 bytes is too small for many
+developer use cases. While there have been some attempts to artificially
+increase the size capacity with address lookup tables, the current cap still is
+artificially limiting the number of signatures and size of other data.
+
+Use cases that are currently limited by the transaction size include:
+
+- ZK proofs such as what is used within [Confidential
+ Balances](https://www.solana-program.com/docs/confidential-balances/zkps)
+- Untruncated Winternitz one-time signatures
+- Nested Multisig used by corporations under Squads
+- Onchain cryptography signature schemes without precompiles such as BLS
+
+and many more. A larger transaction size would also help remove the need for
+address lookup tables, which have been a source of complexity on the validator
+client as well as a frictional developer experience for many developers.
+
+A number of developers have also resorted to using Jito bundles to artificially
+increase the size of their transactions. This is a current workaround that
+developers are using, but it does not have the same guarantees as a transaction
+does on the protocol level, namely that bundles are not guaranteed to be atomic
+like singular transactions.
+
+For these reasons, increasing the current transaction size limit would enable
+developers to create new applications and use cases that were previously
+unfeasible.
+
+## New Terminology
+
+Not at this time.
+
+## Detailed Design
+
+Increasing the transaction size limit above the current MTU max size is
+possible with the use of QUIC. QUIC's RFC 9000 specification does not have
+an explicit maximum stream size, allowing for larger transactions to be
+sent. The larger transaction size would only be supported by the v1 transaction
+format as described in SIMD-0385. Clients will need to support accepting the
+larger transaction sizes on their network ports and not rejest the larger size
```

Comment:
reject -> reject.

Let's also just re-iterate this should only not be rejected for supported tx formats (i.e. not current ones).